main.css

```diff
@@ -1,30 +1,48 @@
 .naugtur .bc-chat-container {
     position: fixed !important;
     z-index: 11511 !important;
-    top: 45px !important;
-    left: 25%;
+    top: 50px !important;
+    right: 200px;
     background: #fff;
-    width: 50% !important;
-    height: calc(96% - 120px) !important;
+    width: 30% !important;
+    height: calc(100% - 110px) !important;
 }
+@media (max-width: 1240px) {
+    .naugtur .bc-chat-container {
+        right: 40px;
+    }
+}
+
+.naugtur #theme-header {
+    display: none !important;
+}
+
+.naugtur #page-wrapper.theme {
+    background: url(http://www.zastavki.com/pictures/originals/2013/Movies___Roger_Pirate_042988_.jpg) top center no-repeat #ddd;
```

Comment:
Adding a bookmarklet has its own security and privacy issues, let's not make it worse by querying another server. It could be github-hosted alongside the main thing (but then it's a copyright issue...)

thoughts?